q2_dada2/plugin_setup.py

```diff
@@ -77,6 +172,13 @@
     inputs={'demultiplexed_seqs':
             SampleData[SequencesWithQuality | PairedEndSequencesWithQuality]},
     parameters={'n': qiime2.plugin.Int},
+    input_descriptions={
+        'demultiplexed_seqs': ('The demultiplexed sequences for which '
+                               'quality plots should be generated.')
+    },
+    parameter_descriptions={
+        'n': 'The number of per-sample quality plots to generate.'
```

Comment:
Add `Twice the number of plots are generated in the case of paired-end sequences.`